### PR TITLE
feat(contracts): revert zero timelock protocol

### PR DESCRIPTION
**Pull request by kangsorang**
# Description

Revert https://github.com/kroma-network/kroma/pull/167, remove protocol for using custom time-lock zero delay for urgent situations.
The default Timelock delay has also been changed to 7 days.

### CODE REVIEW

**Review by 0xHansLee — COMMENTED** [review]
Could you update our spec docs as well?

File: specs/contract-upgrades.md [hunk 1]
```diff
@@ -16,15 +15,7 @@
...
-
-There are two primary types of contract upgrades:
-
-- 30-Day Timelock Delay Upgrade
-- Emergency Upgrade
-
-Typically, when a proposal for an upgrade is approved, it undergoes a mandatory 30-day timelock delay period before
-execution. However, in cases where an urgent upgrade is necessary due to security concerns, the Security Council
-retains the capability to conduct an emergency upgrade.
+When a proposal for an upgrade is approved, it undergoes a mandatory 30-day timelock delay period before execution.
```
Comment (0xHansLee) [inline specs/contract-upgrades.md:18]:
```suggestion
When a proposal for an upgrade is approved, it undergoes a mandatory 7-day timelock delay period before execution.
```

File: packages/contracts/contracts/governance/TimeLock.sol [hunk 2]
```diff
@@ -2,8 +2,8 @@
 pragma solidity ^0.8.15;
 
 import "@openzeppelin/contracts-upgradeable/proxy/utils/Initializable.sol";
+import "@openzeppelin/contracts-upgradeable/governance/TimelockControllerUpgradeable.sol";
```
Comment (seolaoh) [inline packages/contracts/contracts/governance/TimeLock.sol:5]: please sort the imports

File: specs/contract-upgrades.md (hunk 1 above)
Comment (seolaoh) [reply specs/contract-upgrades.md:18]: Please modify `30-day` also in L24.

File: specs/security-council.md [hunk 3]
```diff
@@ -61,6 +61,3 @@ All contract upgrades deployed on [Layer 1][g-l1] are conducted by the
 [governance of the Security Council](contract-upgrades.md#upgrade-by-governance). These upgrades are proposed by a
 member of Security Council and are determined through the voting of these members. If a proposal passes, it typically
 has a 30-day timelock delay for execution.
```
Comment (seolaoh) [inline specs/security-council.md:63]:
```suggestion
has a 7-day timelock delay for execution.
```

**Review by 0xHansLee — APPROVED** [review]
LGTM

**Review by seolaoh — APPROVED** [review]
LGTM

**Pull request merged**